### ISSUE TITLE
Add another platform to contest platforms

### ISSUE THREAD

**Issue by mridul549** [roshan1337d/coding-contests-companion#24]
I want to add Coding Ninjas to the contest platforms list as they also regularly conduct contests.

**Comment by roshan1337d** [comment]
@mridul549 Assigned this to you! Good luck 🚀

**Comment by mridul549** [comment]
thank you for assigning me this issue, I wasn't able to find the Coding Contests Companion channel on the GSSOC server, can you let me know where can I find it?

**Comment by roshan1337d** [comment]
@mridul549 go to the self-roles channel and click the 0 emoji in the message sent on 21/05/2023 at 18:50

**Comment by Shantanusharnagat** [comment]
I would like to work on the issue, can I?

**Comment by mridul549** [comment]
> I would like to work on the issue, can I?

No public API is available for coding ninjas, so we won't be able to add it to the bot.

**Comment by mridul549** [comment]
Yes, will definitely work on it, was not able to find the API before (thanks for the same btw).
